kakoune.tera

```diff
@@ -0,0 +1,69 @@
+---
+whiskers:
+  version: 2.1.0
+  matrix:
+    - flavor
+  filename: "colors/catppuccin_{{ flavor.identifier }}.kak"
+---
+
+{%- set palette = flavor.colors -%}
+
+# Catppuccin {{ flavor.name }}
+
+evaluate-commands %sh{
+    {%- for _, color in palette %}
+    {{ color.identifier }}='rgb:{{ color.hex }}'
+    {%- endfor %}
+
+    echo "
+        set-face global title  ${text}+b
+        set-face global header ${subtext0}+b
+        set-face global bold   ${maroon}+b
+        set-face global italic ${maroon}+i
+        set-face global mono   ${green}
+        set-face global block  ${sapphire}
+        set-face global link   ${blue}
+        set-face global bullet ${peach}
+        set-face global list   ${peach}
+
+        set-face global Default            ${text},${base}
+        set-face global PrimarySelection   ${text},${surface2}
+        set-face global SecondarySelection ${text},${surface2}
+        set-face global PrimaryCursor      ${crust},${rosewater}
+        set-face global SecondaryCursor    ${text},${overlay0}
+        set-face global PrimaryCursorEol   ${surface2},${lavender}
+        set-face global SecondaryCursorEol ${surface2},${overlay1}
+        set-face global LineNumbers        ${overlay1},${base}
+        set-face global LineNumberCursor   ${rosewater},${surface2}+b
+        set-face global LineNumbersWrapped ${rosewater},${surface2}{% if not flavor.dark %}+i{% endif %}
```

Comment:
Not sure what the `+i` thing is but since it was only present for latte I'm assuming it is light/dark related and kept it.